tests/lib/rules-preprocessor/gjs-gts-parser-test.js

```diff
@@ -761,4 +761,80 @@ describe('multiple tokens in same file', () => {
     expect(resultErrors[2].message).toBe("'bar' is not defined.");
     expect(resultErrors[2].line).toBe(17);
   });
+
+  it('lints while being type aware', async () => {
+    const eslint = new ESLint({
+      ignore: false,
+      useEslintrc: false,
+      plugins: { ember: plugin },
+      overrideConfig: {
+        root: true,
+        env: {
+          browser: true,
+        },
+        plugins: ['ember'],
+        extends: ['plugin:ember/recommended'],
+        overrides: [
+          {
+            files: ['**/*.gts'],
+            parser: 'eslint-plugin-ember/gjs-gts-parser',
+            parserOptions: {
+              project: './tsconfig.eslint.json',
+              tsconfigRootDir: __dirname,
+              extraFileExtensions: ['.gts'],
```

Comment:
@patricklx Since we are monkey patching `gts` as `.mts` we probably don't need `extraFileExtensions`.
Also because of our emulation when ppl write `tsconfig.json` it won't work with
```
    "include": [
      "**/*.gts"
    ]
```
I think they will probably need to write
```
    "include": [
      "**/*.mts"
    ]
```
 instead if I'm not mistaken

Reply:
i fixed it now, no need to add .mts extension anymore